opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -765,6 +765,18 @@ def save_as_pt(
 
         # save tokenizer.json in save_json_folder_name
         model.save(save_json_folder_path)
+        with open(save_json_folder_path + "/tokenizer.json") as user_file:
```

Comment:
Can we use `os.path.join` instead? String concat will fail if the user put `/` after the folder name, but `os.path.join` will not.

Reply:
yeah, will be fixed

Reply:
You can add a line to join path and use this path for both read and write.

Reply:
Done